### DISCUSSION TITLE
Need to run the patrol tests on different browsers. Is this possible?

### DISCUSSION THREAD

**Discussion by RashmiShashindra** [leancodepl/patrol#1659] [Q&A]
Hello,

I have started working with Patrol for E2E Integration Tests. 

Now I am looking to run the tests in the chrome/safari and would like to know how to run the tests in browser.

Appreciate any help on this :)

**Comment by RashmiShashindra** [comment]
Hi @bartekpacia,

Appreciate your input on this :) Thanks in advance!

**Comment by bartekpacia** [comment]
Patrol supports only mobile. You can open other apps (like browsers) using `$.native.openApp`. From there on you can use `$.native.tap()` and `$.native.enterText()` to interact with browser's contents.

**Reply by RashmiShashindra** [reply]
@bartekpacia Thank you! Just a quick question. Are you planning to make the patrol to be able to run on web as well? Or is there a way I can use current Patrol tests to run on web builds such as PWAs?

**Reply by bartekpacia** [reply]
It's possible in the future, but we don't have any immediate plans. Please see #1489 for an existing discussion. We're collecting a list of use cases for Patrol Web in there - if you posted your own use case, it'd be great.
